content/04.model-complexity.regularization.md

```diff
@@ -0,0 +1,55 @@
+
+Any supervised learning implementation in rare disease would require robustness towards feature selection from a small number of samples. 
+This robustness is mostly acquired through the combination of various regression strategies. 
+Since machine learning applications in rare disease is infrequent, combination strategies used for rare variant discovery and immune cell signature discovery can serve as good case studies to examine. 
+Many deleterious genomic variants can be extremely rare due to the constant selection pressure working against them. 
+Since the frequency of a rare variant is so low (less than 1%) applying routine statistical procedures that were extensively developed for common variant association, to analyze a low minor allele frequency (MAF) seem inappropriate [@doi:10.1038/nrg2867]. 
+For its feature selection attribute, LASSO or least absolute shrinkage and selection operator has been widely applied in microarray and GWAS data for common variants. 
+But since LASSO by itself is too stringent for rare variants, it has been employed along with group penalties to help identify rare variants/ low frequency predictors [@doi:10.1093/bioinformatics/btq448]. 
+Variations of LASSO have also been implemented to aggregate or group the occurrance of rare variants together by gene or chromosome location [@doi:10.1002/gepi.21746; @doi:10.1186/1753-6561-5-S9-S100; @doi:10.1016/j.ajhg.2008.06.024]. 
+In this strategy, a 0–1 dummy variable was created for each SNP based on the presence or absence of the rare variant. 
+Then linear combinations of the selected dummy variables were considered by using the LASSO procedure. 
+Even though most of the dummy variables were 0, their linear combination was more likely to be nonzero thus leading to increased signal to noise ratio for the rare variants. 
+Only those linear combinations that were nonzero in at least 5% of the subjects were then included to ensure that the new markers were not rare [@doi:10.1186/1753-6561-5-S9-S113; @doi:10.1186/1753-6561-5-S9-S100]. 
+While ridge regression is not usually utilized for feature selection, adaptive ridge regression has been utilized to help combine rare variants into a single score analogous to feature engineering for increasing the signal of rare variants[@doi:10.1371/journal.pone.0044173]. 
+Another variation of LASSO included its integration with the probabilistic logistic bayesian approach to identify a protective rare variant in lung cancer[@doi:10.4137/CIN.S17290]. 
+Xu et al. on the other hand combined the feature selection methods with a generalized pooling strategy, and evaluated the performance of these hybrid approaches for detection of rare genetic variants[@doi:10.1371/journal.pone.0041694]. 
+Another interesting approach is the sparse-group LASSO approach which incorporates prior knowledge into the regularization[@doi:10.1080/10618600.2012.681250]. 
```

Comment:
I wonder if this should be here or the "prior knowledge" section

Reply:
thats a good point, will keep it here for now... but may need to reposition when we put the sections together.